.github/workflows/bridge_history_api.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.20.x
+        go-version: 1.22.x
```

Comment:
Why upgrade the go-version to 1.22?

Reply:
mainly to get access to `slices` package, I can work around it if you would rather staying on go 1.20

Reply:
Got it. Just a quick question. I think we can keep it for now and wait for more reviews.

Reply:
> mainly to get access to slices package, I can work around it if you would rather stay on go 1.20

which function do you want to use ?

Reply:
upgrade to 1.22 is also ok, i think

Reply:
the slices.Index is add in go1.21.0, can switch to this version

Reply:
Sure, will switch to 1.21 :+1:

Reply:
Done